components/camel-any23/pom.xml

```diff
@@ -77,5 +77,16 @@
       <groupId>org.eclipse.rdf4j</groupId>
       <artifactId>rdf4j-runtime</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.camel</groupId>
+      <artifactId>camel-http</artifactId>
+      <scope>test</scope>
+      <version>${project.version}</version>
```

Comment:
project version is not needed here.

Reply:
Hi @oscerd 
When I remove that line I get this error.

`[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.apache.camel:camel-http:jar is missing. @ org.apache.camel:camel-any23:[unknown-version], /home/cedia/beto/camel/components/camel-any23/pom.xml, line 85, column 17
`

Do you know if I have to change something else in order to import other component into a data format project in the test scope ( import camel-http in came-any23).